pandas/tests/io/formats/test_to_csv.py

```diff
@@ -9,6 +10,30 @@
 
 class TestToCSV(object):
 
+    @pytest.mark.xfail((3, 6, 5) > sys.version_info >= (3, 5),
+                       reason='Python csv library bug')
```

Comment:
can you put a refernce to the bug, and a comment on what it is

Reply:
Done.

Reply:
can you also add the link to the xfail reason